base_manifest_extension/hooks.py

```diff
@@ -36,6 +47,31 @@ def _handle_depends_if_installed(cr, manifest):
     depends.extend(module for module, in cr.fetchall())
 
 
+def _handle_rdepends_if_installed(cr, manifest, current_module):
+    graph = _get_graph()
+    if not graph:
+        return
+
+    cr.execute(
+        'select name from ir_module_module '
```

Comment:
Totally stylistic and you'll rarely use it, but please get into the habit of capitalizing SQL commands:

```
SELECT name FROM ir_module_module WHERE state IN %s AND name IN %s
```

Reply:
Sure thing. I prefer this as well but somehow forgot to make the change before committing.